### PR TITLE
ngc_tcp_test: Implement IPsec full offload support

### PR DESCRIPTION
**Pull request by RoiZilberzwaig**
Additional Changes:
- Updated pass criteria for IPsec.
- Modified Collect_BW function to end with return instead of exit.

### CODE REVIEW

File: common.sh [hunk 1]
```diff
@@ -1070,14 +1070,15 @@ get_bandwidth_from_combined_files(){
 
 collect_BW() {
     totalBW=0
+    [ -n "${PASS_CRITERIA}" ] || PASS_CRITERIA=0.9
```
Comment (blochl) [inline common.sh:1073]: Criteria is plural. Criterion is single.

File: common.sh [hunk 2]
```diff
@@ -1110,16 +1111,15 @@ collect_BW() {
     then
         echo "Total throughput in systems is ${totalBW}Gb/s"
     fi
-
```
Comment (blochl) [inline common.sh:1113]: Not related.

File: common.sh (hunk 1 above)
Comment (RoiZilberzwaig) [reply common.sh]: Fixed!

File: common.sh (hunk 2 above)
Comment (RoiZilberzwaig) [reply common.sh]: Fixed!

**Review by blochl — APPROVED** [review]
Thanks! :)

**Pull request merged**